automates/text_reading/src/main/scala/org/clulab/aske/automates/data/DataLoader.scala

```diff
@@ -123,8 +131,11 @@ class MarkdownTextDataLoader extends DataLoader {
     * @param f the File being loaded
     * @return string content of file (wrapped in sequence)
     */
-  def loadFile(f: File): Seq[String] = getTextFromFile(f).split("\n").filter(_.nonEmpty)
+  def loadFile(f: File): Seq[String] = getTextFromFile(f).split("\n").filter(_.nonEmpty).map(t=>t.replace("`", ""))
   override val extension: String = "md"
+  // split on #
```

Comment:
Are these notes needed? They don't seem to explain anything. If they're for the future reference, make the notes into todo or fixme??

Reply:
deleted